optuna/_convert_positional_args.py

```diff
@@ -18,6 +21,14 @@
     _T = TypeVar("_T")
 
 
+_DEPRECATION_WARNING_TEMPLATE = (
+    "Positional arguments {positional_arg} in {func_name}() have been deprecated since v{d_ver}. "
+    "They will be removed in v{r_ver}. "
+    "Please use keyword arguments instead. "
```

Comment:
[nit] They will NOT be removed, but simply updated so that only keyword argument specification works.
```suggestion
    "They will be replaced with the corresponding keyword arguments in v{r_ver}, so please use keyword specification instead. "
```